### PR TITLE
View wallet

### PR DESCRIPTION
**Pull request by Regedit-msc**
Screenshot 2024-04-02 at 13 19 52

View wallet by:
- Using `/home` and clicking the View wallet button
- Using `/view_wallet`

### CODE REVIEW

**Comment by Regedit-msc** [comment]
@Phenzic  I already integrated your code in this pr.

**Review by Phenzic — COMMENTED** [review]
for the explorer_api_base_url is it for making external API requests of for another feature you just added

**Comment by Regedit-msc** [comment]
> for the explorer_api_base_url is it for making external API requests of for another feature you just added

Yes it's to query an API.

File: package.json
Comment (Phenzic) [inline package.json:1]: seeing Axios here I'm  guessing it is for an external request, how about orbs-network/ton-access
If what you are using it for is something the stonFi SDK already offers  we could just go with the one that carries more functionalities, also cuz stonFi SDK will take on a lot of jetton related tasks cc: this week's tasks

File: src/bot/handlers/callbackQuery.ts
Comment (Phenzic) [inline src/bot/handlers/callbackQuery.ts:1]: this clears out a lot, thanks

**Review by Phenzic — COMMENTED** [review]
for the explorer_api_base_url is it for making external API requests of for another feature you just added

File: package.json
Comment (Regedit-msc) [reply package.json:1]: ORBS provides Ton Access a RPC provider with a more lenient limit than Toncenter which we were using before.
You can read up on them on [here](https://www.orbs.com/ton-access).

File: src/bot/handlers/index.ts
[the remark this answers is not in the document]
Comment (Regedit-msc) [reply src/bot/handlers/index.ts:1]: I didn't remove it though.

**Pull request merged**